contracts/src/v0.8/ccip/test/pools/USDC/USDCTokenPool.lockOrBurn.t.sol

```diff
@@ -0,0 +1,201 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
The folder name should match the contract name, so not USDC but `USDCTokenPool` inside the USDC folder. It should follow the source code structure which means the hybrid USDC stuff and this should be in a USDC folder